boefjes/boefjes/plugins/kat_webpage_capture/main.py

```diff
@@ -68,7 +81,7 @@ def get_file_from_container(container: docker.models.containers.Container, path:
 
 def build_playwright_command(webpage: str, browser: str, tmp_path: str) -> str:
     """Returns playwright command including webpage, browser and locations for image, har and storage."""
-    return " ".join(
+    x = " ".join(
```

Comment:
```suggestion
    command = " ".join(
```

Reply:
I'll remove this, was a remnant of my debugging session.